src/clp_logging/handlers.py

```diff
@@ -302,6 +312,12 @@ def update(self, loglevel: int, log_timestamp_ms: int, log_fn: Callable[[str], N
         self.soft_timeout_thread.start()
 
 
+def _get_mutex_context_from_loglevel_timeout(
+    loglevel_timeout: Optional[CLPLogLevelTimeout],
+) -> AbstractContextManager[bool | None]:
```

Comment:
```suggestion
) -> AbstractContextManager[Optional[bool]]:
```
Let's use `Optional` for consistency purposes.